src/mol-util/data-source.ts

```diff
@@ -7,21 +7,14 @@
  * Adapted from LiteMol
  */
 
+import * as fs from 'fs';
```

Comment:
Importing a node module like this causes issues in the browser and forces you to handle it with a bundler like webpack. Please find another way.

Reply:
OK, now importing 'fs' and 'path' via LazyImports